iml-system-rpm-tests/README.md

```diff
@@ -0,0 +1,42 @@
+# IML System RPM Tests
+
+The system level RPM integration tests focus on the creation of various forms of RPM-based lustre filesystems. Vagrant is used to launch the following virtual machines:
+
+1. iscsi - Used for filesystem devices
+1. adm - The manager node
+1. mds[1,2] - The MDS nodes
+1. oss[1,2] - The OSS nodes
+
+## Installation
+
+Install the following software:
+
+1. cargo
+1. rust
+1. sos
```

Comment:
Is sos needed on the host?

Reply:
That's a good question. I took this from the jenkins pipeline but thinking about it more I don't think it is. Maybe I should remove it from the pipeline as well.

Reply:
Since everything runs in the VM, I think we don't need it on the host.